hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx

```diff
@@ -86,7 +88,8 @@ const Orders = () => {
                         </Grid>
                         <Grid item lg={12}>
                             <Grid container spacing={1} direction="row">
-                                {allOrders.map((order, idx) => (
+                                <OrdersTable ordersData={mockPendingOrders} />
```

Comment:
used mockPendingOrders for now, but will later be replaced with `allOrders` from redux